packages/dev/inspector/src/components/actionTabs/tabs/propertyGrids/animations/animationGroupPropertyGridComponent.tsx

```diff
@@ -73,9 +75,9 @@ export class AnimationGroupGridComponent extends React.Component<IAnimationGroup
     updateCurrentFrame(animationGroup: AnimationGroup) {
         const targetedAnimations = animationGroup.targetedAnimations;
         if (targetedAnimations.length > 0) {
-            const runtimeAnimations = animationGroup.targetedAnimations[0].animation.runtimeAnimations;
-            if (runtimeAnimations.length > 0) {
-                this.setState({ currentFrame: runtimeAnimations[0].currentFrame });
+            const runtimeAnimation = animationGroup.targetedAnimations[0].animation.runtimeAnimations.find((rA) => rA.target === animationGroup.targetedAnimations[0].target);

```

Comment:
NIT: can replace both animationGroup.targetedAnimations by targetedAnimations in this line